plugin.rb

```diff
@@ -270,43 +276,43 @@ def include_event_going_total?
         event_start = event['start'] ? event['start'].to_datetime.to_i : nil
-        tc.record_change('event_start', tc.topic.custom_fields['event_start'], event_start)
-        tc.topic.custom_fields['event_start'] = event_start
+        start_change = tc.record_change('event_start', tc.topic.custom_fields['event_start'], event_start)
+        tc.topic.custom_fields['event_start'] = event_start if start_change
 
         event_end = event['end'] ? event['end'].to_datetime.to_i : nil
-        tc.record_change('event_end', tc.topic.custom_fields['event_end'], event_end)
-        tc.topic.custom_fields['event_end'] = event_end
+        end_change = tc.record_change('event_end', tc.topic.custom_fields['event_end'], event_end)
+        tc.topic.custom_fields['event_end'] = event_end  if end_change
 
         all_day = event['all_day'] ? event['all_day'] === 'true' : false
-        tc.record_change('event_all_day', tc.topic.custom_fields['event_all_day'], all_day)
-        tc.topic.custom_fields['event_all_day'] = all_day
+        all_day_change = tc.record_change('event_all_day', tc.topic.custom_fields['event_all_day'], all_day)
+        tc.topic.custom_fields['event_all_day'] = all_day if all_day_change
 
         timezone = event['timezone']
-        tc.record_change('event_timezone', tc.topic.custom_fields['event_timezone'], timezone)
-        tc.topic.custom_fields['event_timezone'] = timezone
+        timezone_change = tc.record_change('event_timezone', tc.topic.custom_fields['event_timezone'], timezone)
+        tc.topic.custom_fields['event_timezone'] = timezone if timezone_change
 
         rsvp = event['rsvp'] ? event['rsvp'] === 'true' : false
-        tc.record_change('event_rsvp', tc.topic.custom_fields['event_rsvp'], rsvp)
-        tc.topic.custom_fields['event_rsvp'] = rsvp
+        rsvp_change = tc.record_change('event_rsvp', tc.topic.custom_fields['event_rsvp'], rsvp)
+        tc.topic.custom_fields['event_rsvp'] = rsvp if rsvp_change
 
         if rsvp
           going_max = event['going_max'] ? event['going_max'].to_i : nil
-          tc.record_change('event_going_max', tc.topic.custom_fields['event_going_max'], going_max)
-          tc.topic.custom_fields['event_going_max'] = going_max
+          going_max_change = tc.record_change('event_going_max', tc.topic.custom_fields['event_going_max'], going_max)
+          tc.topic.custom_fields['event_going_max'] = going_max if going_max_change
 
           going = event['going'] ? event['going'].join(',') : ''
-          tc.record_change('event_going', tc.topic.custom_fields['event_going'], going)
-          tc.topic.custom_fields['event_going'] = going
+          going_change = tc.record_change('event_going', tc.topic.custom_fields['event_going'], going)
+          tc.topic.custom_fields['event_going'] = going if going_change
+        end
+
+        if start_change || end_change || timezone_change # increment by 1, even if more than one props are changed at once
```

Comment:
Doesn't work if `event_version` field is nil. Serves the purpose of preventing an error